code/lib/preview-api/src/modules/store/csf/portable-stories.ts

```diff
@@ -24,23 +25,34 @@ import { normalizeComponentAnnotations } from './normalizeComponentAnnotations';
 import { getValuesFromArgTypes } from './getValuesFromArgTypes';
 import { normalizeProjectAnnotations } from './normalizeProjectAnnotations';
 
-let globalProjectAnnotations: ProjectAnnotations<any> = {};
+let globalProjectAnnotations: FlexibleProjectAnnotations<any> = {};
 
 export function getPortableStoryWrapperId(storyId: string) {
   return `storybook-story-${storyId}`;
 }
 
+function extractAnnotation<TRenderer extends Renderer = Renderer>(
+  annotation: FlexibleProjectAnnotations<TRenderer>
+) {
+  // support imports such as
+  // import * as annotations from '.storybook/preview'
+  // in both cases: 1 - the file has a default export; 2 - named exports only
```

Comment:
nit: feel like there's missing a line here. Referring to "both cases" but only one case is described.